README.md

```diff
@@ -1,2 +1,73 @@
-# CORE
 [![Build Status](https://travis-ci.org/mentorpaired/core.svg?branch=staging)](https://travis-ci.org/mentorpaired/core) [![Coverage Status](https://coveralls.io/repos/github/mentorpaired/core/badge.svg?branch=staging)](https://coveralls.io/github/mentorpaired/core?branch=staging)
+
+# CORE
+Core is the backend of MentorPaired, an app which connects software engineering mentors to mentees who seek to learn any programming language.
+
+### Tech
+Core is written in Python3 and Django 3.0.3.
+
+### Installation For Ubuntu 18.04 Users
+
+ Please install and set up the following packages first, upgrade if you find the package is already installed:
+* Python3. Run the 'python3 -V' command to see the version you have installed.
+* PostgreSQL (Ensure the server is running). Alternatively, if your project is a small one or you find you don't need PostgreSQL, Python includes a lightweight database called SQLite so you won't need to set up a database, however it cannot support a high level of concurrency.
```

Comment:
Let's all stick to Postgres instead to be consistent. We can scrap the alternative

Reply:
Okay